src/components/text/Text.jsx

```diff
@@ -40,13 +40,41 @@ const WEIGHT = {
   BOLD: 'emphasised'
 };
 
-const Text = ({children, size, weight, color, className, type = TYPE.DIV, noWrap, breakWords, ...props}) => {
+const TEXT_TRANSFORM = {
```

Comment:
@aju TEXT_WEIGHT, TEXT_COLOR, TEXT_SIZE?

Reply:
@aju or it would be too big refactor for now?

Reply:
One step at a time:P